requirements.txt

```diff
@@ -5,4 +5,3 @@ psutil>=5
 scikit-learn>=0.20.0
 fastdtw
 setuptools>=40.1.0
-
```

Comment:
Could you please rollback the changes here since and a empty line in the end as it was before?

Reply:
@adekusar-drl OK, I did that ;)